src/axios/README.md

```diff
@@ -0,0 +1,112 @@
+# <center>Axios</center>
+
+## Importing axios
+
+```js
+import axios from "./axios";
+```
+
+## Example
+
+```js
+import axios from "./axios";
+
+// axios.<method> will now provide autocomplete and parameter typings
+```
+
+#### Performing a `GET` request
+
+```js
+// Make a request for a user with a given ID
+axios
+    .get("/user?ID=12345")
+    .then(function ({ error, response }) { // note the object destructuring
+        if(error) {
+            // handle error
+            console.log(error);
+        } else {
+            // handle success
+            console.log(response);
+        }
+    })
+// Optionally the request above could also be done as'
+axios.get("/user",
+    params: {
```

Comment:
what is this syntax? 
axios.get(url, { ID: 12345 }) // it should be like this what is `params` here